crates/adapters/src/test/mod.rs

```diff
@@ -199,12 +194,12 @@ where
     Ok((reader, consumer, parser, input_handle))
 }
 
-/// Create a simple test circuit that passes the input stream right through to
-/// the output.
-// TODO: parameterize with the number (and types?) of input and output streams.
-pub fn test_circuit<T>(
+/// Create a simple test circuit that passes each of the `n` streams right
+/// through to the corresponding output.
+pub fn test_circuit_multiple<T>(
```

Comment:
In the backfill avoidance PR, `test_circuit` labels all operators with persistent id's, which is necessary for suspend, resume, and recovery to work.

Reply:
I changed `test_circuit` to:

```rust
/// Create a simple test circuit that passes each of a number of streams right
/// through to the corresponding output.  The number of streams is the length of
/// `persistent_output_ids`, which also specifies optional persistent output ids.
pub fn test_circuit<T>(
    config: CircuitConfig,
    schema: &[Field],
    persistent_output_ids: &[Option<&str>],
) -> (DBSPHandle, Box<dyn CircuitCatalog>)
```

so that the most commonly passed persistent_output_ids became `&[None]` for one stream without a persistent id.